airbyte-webapp/src/pages/SourcesPage/pages/CreateSourcePage/components/SourceForm.tsx

```diff
@@ -82,12 +111,13 @@ export const SourceForm: React.FC<SourceFormProps> = ({
       onSubmit={onSubmitForm}
       formType="source"
       availableServices={sourceDefinitions}
+      checkConnectionBeforeSubmit={sourceCloneId ? false : true}
```

Comment:
Hey @timroes yeah because when we clone the source it won't send the airbyte_secrets and they will default to (******). So, when we check the values are always wrong and they will always fail.

Reply:
I will check this during review (but we had some high prio stuff come in, so this might be a bit delayed), but if we have no way to validate the full exact config of the connector we're going to save (which we should), then we might need to add capabilities on the BE for this, before continuing here. I could imagine that the test request might get a new property `secretsFrom: string` which you could use to copy secrets over from an already existing connector for testing.

Reply:
I have added a way to test the config so yeah you can check while you are reviewing it no problem